index.html

```diff
@@ -31,9 +31,53 @@ <h1 class="title">Error</h1>
 };
     </script>
 
-    <script src="node_modules/three/build/three.js"></script>
-    <script src="node_modules/three/examples/js/controls/VRControls.js"></script>
-    <script src="node_modules/three/examples/js/effects/VREffect.js"></script>
-    <script src="build/embed.js"></script>
+    <script>
```

Comment:
My inclination is to avoid inline scripts as much as possible. Could you make this a separate lazy loader script?